### ISSUE TITLE
is it possible to use cachix API keys rather the traditional signing keys?

### ISSUE THREAD

**Issue by nixinator** [hercules-ci/support#60]
As highlighted here.

https://blog.cachix.org/posts/2020-11-09-write-access-control-for-binary-caches/

**Comment by roberth** [comment]
Yes, this is supported. The reference and guide documentation needs to be updated to clarify this.

Cachix caches without signingKeys will be pushed to, as part of the write token feature (Cachix-managed signing keys). Make sure to enter an authToken with write access. `CachixCache` reference doc: https://docs.hercules-ci.com/hercules-ci-agent/binary-caches-json/#_cachixcache

**Comment by nixinator** [comment]
seems to be working well. Here's what i used in my binary-caches.json

```
{ "name-of-the-cachix-cache":
    { "kind": "CachixCache"
    , "authToken": "auth write token generated for the above cache on cachix.org"
    , "publicKeys": ["the public key of the cachix cache"]
    }
}
```
